src/configuration.ts

```diff
@@ -1402,9 +1414,10 @@ export async function initFromStateAndSettings(): Promise<void> {
             }
 
             // Final updates in some constructs that depend on more than one of the above settings.
```

Comment:
configureDirty is about we run or not configure later when querying the right info.
In here, we need to re-analyze configure params and update the tree nodes (the ones about make/makefile/build.log, not about configuration/build-target/launch-target, those get updated via their SET actions... but make/makefile/build.log don't have yet SET actions, they are modified via settings updates).